tests/functional/keep-going.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+source common.sh
+
+clearStore
+
+# XXX: These invocations of nix-build should always return 100 according to the manpage, but often return 1
```

Comment:
Can you create an issue for this?

Reply:
Ok, @iFreilicht is the one who discovered the error. Probably could use more details before creating the issue?

Reply:
Well, that's all I found out, I don't have more details for you 😬 But it basically happened during testing, so you should be able to reproduce that issue and can then create one with a proper log. This might also be why the tests are failing right now.

Reply:
Oh, alright. I did check the blame and it looks like @Ericson2314 did write some of the worker stuff around where I'm touching. Maybe John has some ideas into getting this to pass.